### ISSUE TITLE
AntiDepression feature

### ISSUE THREAD

**Issue by LostNuke** [Discord-FurDevs/FurDevs-Bot#20]
While coding and falling over so many Bugs. Its always good to have a way to get yoself happy. So why not create a feature, that lets you go like. !depressed and it spams you with 10 cat pics or animals that are cute. Alternatively we could have the same, but it spams you with 10 random memes AND animal pics ;D

**Comment by thetayloredman** [comment]
That would be API abuse to send that many messages @LostNuke

**Comment by thetayloredman** [comment]
*in a short period of time

**Comment by ElectricDaAluki** [comment]
Maybe instead of 10 just do like 3 and have them wait 20 seconds. Modern Problems require modern solutions.

**Comment by thetayloredman** [comment]
Bots should only fire one message on command.

**Comment by DeveloperDrago** [comment]
_laughs in blame command_

**Comment by ElectricDaAluki** [comment]
> Bots should only fire one message on command.

Yeah, but i'm sure there are plenty of public bots who doesn't follow that.

**Comment by LostNuke** [comment]
I mean... you all have a great.. BUT wrong point... you can just give an array of attachments in the send command... yall know that right?

**Comment by thetayloredman** [comment]
> spams you
made me think otherwise, sorry!

**Comment by LostNuke** [comment]
Fair enough my bad xD
